umap/static/umap/js/modules/request.js

```diff
@@ -0,0 +1,157 @@
+// Uses `L._`` from Leaflet.i18n which we cannot import as a module yet
+import { Evented, DomUtil } from '../../vendors/leaflet/leaflet-src.esm.js'
+
+export class HTTPError extends Error {
```

Comment:
Should all the exception inherit from the same base class, so we can intercept all the ones from this module alltogether?